warehouse/ingest-core/src/main/java/datawave/ingest/table/aggregator/KeepCountOnlyUidAggregator.java

```diff
@@ -0,0 +1,12 @@
+package datawave.ingest.table.aggregator;
+
+/**
+ * An extension of {@link GlobalIndexUidAggregator} that will always preserve "count only" Uid.List protobufs, even when their count goes below 1. This iterator
+ * will prevent the accidental deletion of count only protobufs in case extra deletions occur.
```

Comment:
Actually the names of things are probably fine here.  In this comment here however explain that seenIgnore means that the Uid.List count exceeded the max.

Reply:
Sounds good. Will update.